qemu/tests/cfg/viorng_in_use.cfg

```diff
@@ -0,0 +1,43 @@
+- viorng_in_use:
+    type = driver_in_use
+    start_vm = yes
+    kill_vm_on_error = yes
+    check_guest_bsod = yes
+    login_timeout = 240
+    suppress_exception = True
+    driver_load_cmd = C:\devcon.exe enable @DRIVER_ID
+    driver_unload_cmd = C:\devcon.exe disable @DRIVER_ID
+    driver_id_cmd = C:\devcon.exe find * | find "VirtIO"
+    run_bgstress = rng_bat
+    target_process = D:\random_%PROCESSOR_ARCHITECTURE%.exe
+    rng_dll_register_cmd = "rundll32 C:\Windows\system32\viorngum.dll,RegisterProvider"
+    session_cmd_timeout = 240
+    read_rng_cmd  = "X:\random_%PROCESSOR_ARCHITECTURE%.exe"
+    enable_verifier_cmd = "verifier.exe /standard /driver viorng.sys"
```

Comment:
I think it's better to add a flag here to check whether need enable verifier here.